content/en/security/identity_risks/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cloud Security Management Identity Risks
+kind: documentation
+further_reading:
+- link: "/security/cloud_security_management/"
+  tag: "Documentation"
+  text: "Learn more about Cloud Security Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+CSM Identity Risks is not available in this site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">CSM Identity Risks is in beta.</div>
+
+Cloud Security Management Identity Risks (CSM Identity Risks) provides in-depth visibility into your organization's AWS IAM risks. It enables you to detect and resolve identity risks on an ongoing basis to secure your cloud infrastructure from IAM-based attacks.
+
+<div class="alert alert-info">At this time, CSM Identity Risks is available for AWS only.</div>
+
+## Setup
+
+To use CSM Identity Risks, you must [enable resource collection for AWS][2] and [enable CloudTrail logs forwarding][4]. If you've already done this, no additional setup is required.
```

Comment:
Can you point the [2] link above to https://docs-staging.datadoghq.com/deforest/csm-identities/integrations/amazon_web_services/?tab=roledelegation#cloud-security-posture-management?